### ISSUE TITLE
loop

### ISSUE THREAD

**Issue by sannedubois** [naomiaro/waveform-playlist#15]
Hi Naomi,

It would be great if your multi track audio player could have a loop-function too. Also it would be nice if a user could specify the amount of repeats using a special button and the part of the audio that has to be repeated. For my students I'm using play-along-tracks which have 4 or 8 metronome clicks. When a student chooses to have the track - let's say - repeated 3 times, it has to repeat/loop the part that comes after the metronomes, because you want to listen to the metronome only once.

Would be great.

Kind regards,
Derk

**Comment by naomiaro** [comment]
Yeah that sounds like a good feature, hopefully I'll have some more time to work on this project soon!
